ftests/nuxeo-platform-explorer-ftests/src/test/java/org/nuxeo/functionaltests/explorer/ITExplorerApidocManagerTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Anahide Tchertchian
+ */
+package org.nuxeo.functionaltests.explorer;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.nuxeo.apidoc.security.SecurityHelper;
+import org.nuxeo.functionaltests.RestHelper;
+import org.nuxeo.functionaltests.explorer.pages.DistribAdminPage;
+import org.nuxeo.functionaltests.explorer.pages.ExplorerHomePage;
+import org.nuxeo.functionaltests.explorer.pages.UploadFragment;
+
+/**
+ * Tests features for {@link SecurityHelper#APIDOC_MANAGERS_GROUP} members.
+ *
+ * @since 11.1
+ */
+public class ITExplorerApidocManagerTest extends ITExplorerAdminTest {
+
+    @Override
+    @Before
+    public void before() {
+        if (!RestHelper.groupExists(SecurityHelper.DEFAULT_APIDOC_MANAGERS_GROUP)) {
+            RestHelper.createGroup(SecurityHelper.DEFAULT_APIDOC_MANAGERS_GROUP, "Apidoc Managers");
+        }
+        if (!RestHelper.userExists(MANAGER_USERNAME)) {
+            RestHelper.createUser(MANAGER_USERNAME, TEST_PASSWORD, null, null, null, null,
+                    SecurityHelper.DEFAULT_APIDOC_MANAGERS_GROUP);
+        }
```

Comment:
Could it be interesting to have a `RestHelper#createUserIfDoesntExist` and the equivalent for the group?

Reply:
Indeed, i finally made that change because it was fed up of adding and removing the line "RestHelper.removeUser" whenever i ended tests abruptly locally (typically killing FF).
I can do this change if you want.

Reply:
done with createUserIfItDoesNotExist, same for groups